Car_Localizer/Car_Localizer/Training/model_vehicle_classifier.py

```diff
@@ -0,0 +1,29 @@
+import os
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
+import torch.nn as nn
+
+#CNN Architecture:
+class VehicleClassifier(nn.Module):
```

Comment:
btw, we have a model already on a branch, but if you like to create new one, please derive from `Model_Base`